prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -74,6 +74,7 @@ data class Hl7Configuration
     val suppressHl7Fields: String? = null,
     val suppressAoe: Boolean = false,
     val defaultAoeToUnknown: Boolean = false,
+    val replaceUnicodeWithAscii: Boolean = false,
```

Comment:
Another name can be replaceDiacritics, let me know your thoughts.

Reply:
I think this name is fine.